src/app/wallets/sidebar.tsx

```diff
@@ -150,13 +150,17 @@ export const sidebar: SideBar = {
 					name: "Get Started",
 					href: `${smartWalletSlug}/get-started`,
 				},
+				{
+					name: "Permissions & Session Keys",
+					href: `${smartWalletSlug}/permissions`,
+				},
 				{
 					name: "Bundler & Paymaster",
 					href: `${smartWalletSlug}/infrastructure`,
 				},
 				{
-					name: "Permissions & Session Keys",
-					href: `${smartWalletSlug}/permissions`,
+					name: "Sponsorship rules",
```

Comment:
'sponsorship rules' or 'sponsorship policies'? i like rules, if we good with that i can change it in dashboard too